### PR TITLE
[SL-2432] feat: Add connect page with redirect, refactor object listing styling

### PR DESCRIPTION
**Pull request by james-wallis**
- Adds `/beta/connect` page to connect to a customer environment easily (saves copying and pasting environment info)
- Changes the `__typename` in Filters to `object type`
- Refactors object listing styling to:
  - Span more of home page
  - Show the end of a Pill when stickied so that its easier to identify the object type

* [x] Feature

https://skylarkplatform.atlassian.net/browse/SL-2432

### CODE REVIEW

File: src/pages/beta/connect.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+import { LOCAL_STORAGE } from "src/constants/skylark";
+import { useConnectedToSkylark } from "src/hooks/useConnectedToSkylark";
+
+export default function BetaConnect() {
+  const { query, push: navigateTo } = useRouter();
+  // const { connected } = useConnectedToSkylark();
```
Comment (freixedelo) [inline src/pages/beta/connect.tsx:9]: want to keep?

File: src/pages/beta/connect.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        LOCAL_STORAGE.betaAuth.token,
+        query.apikey as string,
+      );
+      // storage events are not picked up in the same tab, so dispatch it for the current one
+      window.dispatchEvent(new Event("storage"));
+      navigateTo("/");
+    }
+  }, [query, navigateTo]);
+
+  console.log({ query });
```
Comment (freixedelo) [inline src/pages/beta/connect.tsx:24]: want to keep?

File: src/pages/beta/connect.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  return (
+    <div className="flex h-screen w-screen flex-col items-center justify-center gap-4">
+      <h1 className="mb-4 font-heading text-3xl">Skylark Auto Connect</h1>
+      {!query.uri || !query.apikey ? (
+        <>
+          <p>
+            Enter your Skylark URI and API Key into the URL to auto connect.
+          </p>
+          <p>
+            {"Format: "}{" "}
```
Comment (freixedelo) [inline src/pages/beta/connect.tsx:35]: can be plain text at this point right? (no `{}`)

File: src/constants/skylark.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ export const REQUEST_HEADERS = {
 };
 export const OBJECT_LIST_TABLE = {
   columnIds: {
-    objectType: "__typename", // This is the built-in GraphQL __typename
+    objectType: "object type",
```
Comment (freixedelo) [inline src/constants/skylark.ts:12]: this used for? is the space ok? 😅

Comment (james-wallis) [reply src/constants/skylark.ts:12]: space is good, just used in filters really

File: src/pages/beta/connect.tsx (hunk 3 above)
Comment (james-wallis) [reply src/pages/beta/connect.tsx]: great spot

**Review by freixedelo — APPROVED** [review]

**Pull request merged**